packages/SwingSet/src/kernel/vatAdmin/vatAdmin-src.js

```diff
@@ -34,8 +34,8 @@ export function buildRootDeviceNode({ endowments }) {
       const vatID = kernelVatCreationFn({ bundleName }, options);
       return vatID;
     },
-    terminate(vatID) {
-      kernelTerminateVatFn(vatID);
+    terminate(vatID, message) {
```

Comment:
is this new argument the same `reason` that everything else gets? if so, should it be named `reason` instead?

Reply:
Only the terminate-on-failure value should be called `reason`. The terminal success value never is. Further, the `reason` is strongly expected to be an error object and almost always is. But this has never been an enforced restriction --- the `reason` can in fact be anything.